### PR TITLE
patch serverless.yml with comments and example

### PR DESCRIPTION
**Pull request by ole3021**
This is PR for issue: https://github.com/serverless/roadmap-tencent/issues/775

* add welcome message in serverless.yml
* introduce the serverless components
* add help doc link
* add comments for fields
* add examples to config domain and SSL

### CODE REVIEW

**Comment by zongUMR** [comment]
@yugasun @AprilJC Please take a look at Oliver's changes for components

File: express-starter/src/serverless.yml [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
+    src: ./ # 本地需要打包的文件目录
+    exclude: # 被排除的文件或目录
       - .env
-  region: ap-guangzhou
-  runtime: Nodejs10.15
-  apigatewayConf:
-    protocols:
+      - .git
+  apigatewayConf: # api网关配置
+    enableCORS: false # 是否允许跨域
```
Comment (timqian) [inline express-starter/src/serverless.yml:30]: 这里为什么要阻止用户跨域？

**Review by yugasun — APPROVED** [review]
LGTM

**Review by zongUMR — APPROVED** [review]

**Comment by zongUMR** [comment]
@ole3021 You can merge this PR

File: express-starter/src/serverless.yml (hunk 1 above)
Comment (ole3021) [reply express-starter/src/serverless.yml:30]: cors默认配置是false，这里的主要目的是现实这个选项以提示用户可以通过修改选项来配置cors。 @timqian

**Pull request merged**